compiler/codegen/symbol.cpp

```diff
@@ -965,6 +965,26 @@ GenRet ArgSymbol::codegen() {
   return ret;
 }
 
+static std::string getFortranTypeName(Type* type) {
+  std::string typeName = fortranTypeNames[type->symbol];
+  if (typeName.empty()) {
+    // TODO: Maybe issue an error instead?
```

Comment:
For this an the TODO in getFortranKindName, would it work reasonably well to add a USR_WARN warning for now? It seems that we're concerned the programmer would be surprised. Does your test rely on this behavior?